frontend/packages/network-attachment-definition-plugin/integration-tests/tests/models/nadForm.ts

```diff
@@ -0,0 +1,68 @@
+import { browser, ElementFinder, ExpectedConditions as until } from 'protractor';
+import { BROWSER_TIMEOUT, testName } from '@console/internal-integration-tests/protractor.conf';
+import { createYAMLButton, isLoaded } from '@console/internal-integration-tests/views/crud.view';
+import { PAGE_LOAD_TIMEOUT_SECS } from '@console/kubevirt-plugin/integration-tests/tests/utils/consts';
+import { click, selectDropdownOptionById } from '@console/shared/src/test-utils/utils';
+import * as nadFormView from '../../views/nad.form.view';
+
+// TODO See about moving this function from kubevirt-plugin to console-shared
```

Comment:
We should probably move it to console-shared and simplify the logic as there should be no need for the PF3 workarounds...
It can be done as a followup though as the current implementation works reliably and removing the workarnounds requires proper testing due to the random nature of the issues we had

Reply:
Agreed! I'll add it to the list of followups.